hugolib/shortcode.go

```diff
@@ -206,7 +206,7 @@ type shortcode struct {
 	// Before Hug0 0.55 we didn't send any shortcode output to the markup
 	// renderer, and this flag told Hugo to process the {{ .Inner }} content
 	// separately.
-	// The old behaviour can be had by starting your shortcode template with:
+	// The old behavior can be had by starting your shortcode template with:
```

Comment:
I know I have merged a similar "typo fix" before, but as these comments are mostly written by me, and I write British English, I don't think we can call the above a "typo". It's listed first in the Oxford dictionary (even as the first option):

https://www.oed.com/search/dictionary/?scope=Entries&q=behaviour

Reply:
We could revert that if you prefer, but most English technical documentation is written as American English. I thought it’d be best for it to be consistent.

Reply:
I think you would find that most English technical documentation is written in the language of the people who created it. But OK, I guess one could argue that _some_ of the comments in this repo is also used in the HugoDocs, which we have said should be in Simple English.